benchexec/resources.py

```diff
@@ -15,191 +10,157 @@
 import math
 import os
 import sys
+from functools import cmp_to_key
 
-from benchexec import cgroups
-from benchexec import util
+import cgroups
+import util
 
-__all__ = [
-    "check_memory_size",
-    "get_cpu_cores_per_run",
-    "get_memory_banks_per_run",
-    "get_cpu_package_for_core",
-]
```

Comment:
Why is this removed? It declares which functions in this file are supposed to be called from outside.